### PR TITLE
DDF-3297 Update policy for WFS

### PR DESCRIPTION
**Pull request by jlcsmith**
#### What does this PR do?
Added permissions needed for WFS federated source bundles

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@dcruver @AzGoalie 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
@codice/security 

#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@bdeining
@stustison

#### How should this be tested? (List steps with links to updated documentation)
Verify WFS 1.0, 1.1.0, and 2.0 federated source can be configured.

#### What are the relevant tickets?
[DDF-3297](https://codice.atlassian.net/browse/DDF-3297)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/ddf-common/src/main/resources/security/default.policy [hunk 1]
```diff
@@ -525,4 +534,5 @@ grant {
     permission javax.management.MBeanTrustPermission "register";
 
     permission org.osgi.service.cm.ConfigurationPermission "*", "configure";
+
```
Comment (AzGoalie) [inline distribution/ddf-common/src/main/resources/security/default.policy]: Is this newline necessary?

**Comment by jlcsmith** [comment]
build now

File: distribution/ddf-common/src/main/resources/security/default.policy [hunk 2]
```diff
@@ -390,6 +390,15 @@ grant codeBase "file:/security-sts-attributequeryclaimshandler/security-sts-ldap
...
 
+grant codebase "file:/spatial-wfs-v1_1_0-source" {
+    permission java.lang.RuntimePermission "createClassLoader", "*"};
+
+grant codebase "file:/spatial-wfs-v2_0_0-source" {
+    permission java.lang.RuntimePermission "createClassLoader", "*"};
+
+grant codebase "file:/spatial-wfs-v1_0_0-source" {
+    permission java.lang.RuntimePermission "createClassLoader", "*"};
+
```
Comment (stustison) [inline distribution/ddf-common/src/main/resources/security/default.policy]: Can you please combine all of these to ```grant codebase "file:/spatial-wfs-v1_0_0-source/spatial-wfs-v2_0_0-source/spatial-wfs-v1_1_0-source" {
    permission java.lang.RuntimePermission "createClassLoader";
}```

There should already be a block that has only `permission java.lang.RuntimePermission "createClassLoader";` and if that is the case (I'm pretty sure it is) these bundles should be added to that block. This is for performance reasons. The startup performance hit for the security manager is a quadratic function of how many of these blocks there are. We don't want to create more of them than we need to.

**Comment by jlcsmith** [comment]
Closing to https://github.com/codice/ddf/pull/3251

**Pull request closed without merging**